features/security/src/main/feature/feature.xml

```diff
@@ -86,6 +92,7 @@
 
     <feature name="web-container" version="${project.version}">
         <feature prerequisite="true">platform-filter-delegate</feature>
+        <feature prerequisite="true">platform-error-page-injector</feature>
```

Comment:
❓ Why do we need prerequisite="true" here? If it is needed, can we put a comment stating why it is needed?

Reply:
I believe it is needed, just as it was needed for the `platform-filter-delegate` because it needs to come up first. We can add a comment explaining it. That being said, we will look more closely into the ordering of these features/bundles soon since we have seen flaky ordering issues recently with `platform-filter-delegate` which led to https://github.com/codice/ddf/pull/3581.

Reply:
I made the assumption that web-container started up servlets that would need an error page added to them. @tbatie is this not the case?